.github/workflows/semantic-checker.yml

```diff
@@ -0,0 +1,54 @@
+name: MageOS Semantic Checker
+run-name: ${{ github.actor }} is checking the semantic version
+on: [pull_request]
+jobs:
+  semantic-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Environment Debugging Output
+        run: |
+          echo "Event_path: ${GITHUB_EVENT_PATH}"
+          echo "Event: ${{ github.event }}"
+      - name: Check out Semamtic Version Checker tool
+        uses: actions/checkout@v3
+        with:
+          repository: magento/magento-semver
```

Comment:
I've forked the repo into the mage-os project: https://github.com/mage-os/mirror-magento-semver
Can we use our fork instead of the upstream one?

Reply:
Yes, I will change this to refer to the MageOS mirror of SVC.

Reply:
Workflow now uses the MageOS mirror of SVC

Reply:
Thank you